fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/A320_251N/A320Payload.tsx

```diff
@@ -452,6 +460,23 @@ export const A320Payload: React.FC<A320Props> = ({
         gsxPayloadSyncEnabled,
     ]);
 
+    // If totalPax is 0, deactivate all sounds from the cabin.
+    useEffect(() => {
+        const cabinSoundStatus:number = (totalPax > 0 ? 1 : 0);
+
+        if (globalSettingsRedux.passengerAmbienceActive) {
+            setPassengerAmbienceEnabled(cabinSoundStatus);
+        }
+
+        if (globalSettingsRedux.cabinAnnouncementsActive) {
+            setAnnouncementsEnabled(cabinSoundStatus);
+        }
+
+        if (globalSettingsRedux.boardindgMusicActive) {
```

Comment:
```suggestion
        if (globalSettingsRedux.boardingMusicActive) {
```

(But I think this typo was due to no autocompletion from a lack of typing as I outlined above)

Reply:
Typing error by me.